src/routes/transactions/entities/transaction-data.entity.ts

```diff
@@ -28,7 +28,7 @@ export class TransactionData {
     hexData: string | null,
     dataDecoded: DataDecoded | null,
     to: AddressInfo,
-    value: string,
+    value: string | null,
```

Comment:
As we weren't inferring the types from the schema, this was not caught. It matches the types of the Transaction Service.